internal/characters/xiao/burst.go

```diff
@@ -30,13 +30,14 @@ func (c *char) Burst(p map[string]int) action.ActionInfo {
 	// TODO: No indication of that in library though
 	c.AddStatus(burstBuffKey, 900+burstStart, true)
```

Comment:
I find it a bit strange that the burst buff starts as soon as he casts. So in this case, does the A1 buff start ticking on cast too?

Reply:
yes